packages/compass-collection/src/stores/context.tsx

```diff
@@ -360,19 +371,7 @@ const createContext = ({
   const globalAppRegistry = state.appRegistry;
   const roles = globalAppRegistry.getRole('Collection.Tab') || [];
 
-  // Filter roles for feature support in the server.
-  const filteredRoles = roles.filter((role: any) => {
-    if (
-      ['Indexes', 'Validation', 'Explain Plan'].includes(role.name) &&
-      isDataLake
-    ) {
-      return true;
-    }
-    if (!role.minimumServerVersion) return true;
-    return semver.gte(serverVersion, role.minimumServerVersion);
```

Comment:
Here's where we were previously using `semver` which we don't need anymore. No plugins were using `minimumServerVersion` in their role definition.